src/qrencode.mk

```diff
@@ -0,0 +1,40 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := qrencode
+$(PKG)_WEBSITE  := https://fukuchi.org/works/qrencode/
+$(PKG)_DESCR    := libqrencode
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 3.4.4
+$(PKG)_CHECKSUM := e794e26a96019013c0e3665cb06b18992668f352c5553d0a553f5d144f7f2a72
+$(PKG)_SUBDIR   := qrencode-$($(PKG)_VERSION)
+$(PKG)_FILE     := qrencode-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://fukuchi.org/works/qrencode/qrencode-$($(PKG)_VERSION).tar.gz
+$(PKG)_DEPS     := gcc libpng
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- 'https://fukuchi.org/works/qrencode/index.html' | \
+    grep 'qrencode-' | \
+    $(SED) -n 's,.*qrencode-\([0-9][^>]*\)\.tar.*,\1,p' | \
+    $(SORT) -Vr | \
+    head -1
+endef
+
+define $(PKG)_BUILD
+    cd '$(BUILD_DIR)' && $(SOURCE_DIR)/configure \
+        CFLAGS='-pthread' \
+         --without-tools \
+        $(MXE_CONFIGURE_OPTS) \
+        CONFIG_SHELL=$(SHELL)
+    $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
+    $(MAKE) -C '$(BUILD_DIR)' -j 1 install
+
+    # create pkg-config files
+    $(INSTALL) -d '$(PREFIX)/$(TARGET)/lib/pkgconfig'
+    (echo 'Name: $(PKG)'; \
+     echo 'Version: $($(PKG)_VERSION)'; \
+     echo 'Description: libqrencode'; \
+     echo 'Libs: -lqrencode';) \
+     > '$(PREFIX)/$(TARGET)/lib/pkgconfig/$(PKG).pc'
+
```

Comment:
Add a test, please.

Reply:
I did update the .mk build command to run --with-tests , which builds the test exes. This was enough to identify a linking problem with libpng, which i fixed by adding a dependency and -lpng. 

However, i'm not sure how to run the test exe binaries in the mxe environment on Linux. 

If i add a line like this:
`$(PREFIX)/$(TARGET)/bin/test_qrencode.exe`

... i get this:
```
'/home/raulg/cc/utils/mxe/tmp-qrencode-i686-w64-mingw32.static/qrencode-3.4.4.build_'/tests/test_qrencode.exe
bash: /home/raulg/cc/utils/mxe/tmp-qrencode-i686-w64-mingw32.static/qrencode-3.4.4.build_/tests/test_qrencode.exe: cannot execute binary file: Exec format error
make[1]: *** [build-only-qrencode_i686-w64-mingw32.static] Error 126
```